include/cantera/base/NoExitLogger.h

```diff
@@ -0,0 +1,26 @@
+#ifndef NOEXITLOGGER_H
+#define NOEXITLOGGER_H
+
+#include <string>
+#include "cantera/base/logger.h"
+
+using namespace std;
+
+static std::string ss = "print \"\"\" ";
+
+namespace Cantera {
+
+  /// Logger that doesn't exit when an error is thrown.
+  /// @ingroup textlogs
+  class NoExitLogger : public Logger {
+  public:
+    NoExitLogger() {}
```

Comment:
Cantera's style is not to indent inside a `namespace` directive, and then to use 4 spaces for each level after that.